renderer.go

```diff
@@ -0,0 +1,55 @@
+package survey
```

Comment:
Okay, last thing before we get this in - do you mind moving this inside of `core`? I know its not very conventional, but I tend to prefer internal packages with public exports to having a large flat codebase with internal types/variables.